server/game/cards/characters/09/macetyrell.js

```diff
@@ -0,0 +1,43 @@
+const DrawCard = require('../../../drawcard.js');
+
+class MaceTyrell extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                onCardEntersPlay: event => event.card.isFaction('tyrell') && event.card.getType() === 'character' &&
+                                           event.card.controller === this.controller
+            },
+            limit: ability.limit.perPhase(1),
+            cost: ability.costs.payGold(1),
+            handler: () => {
+                this.modifyPower(1);
+                this.game.addMessage('{0} pays 1 gold to gain 1 power on {1}', this.controller, this);
+            }
+        });
+
+        this.action({
+            title: 'Remove character from game',
+            cost: ability.costs.kneelFactionCard(),
+            target: {
+                activePromptTitle: 'Select a character',
+                cardCondition: card => card.location === 'play area' && card.controller === this.controller &&
```

Comment:
Technically, the card doesn't say that it has to be a Tyrell character you control, only that it's another Tyrell character. Unless I'm misreading or missing a rule, it seems like you could use the ability offensively by removing your opponent's heavy hitting Tyrell character from the game at the start of the challenges phase.

(if that's correct, it means we need a saveable version of "remove from game" in case such a card has a duplicate on it)